pkg/commands/artifact/run.go

```diff
@@ -118,12 +118,12 @@ func NewRunner(ctx context.Context, cliOptions flag.Options, opts ...RunnerOptio
 		opt(r)
 	}
 
-	// If the user has not disabled notices or is running in quiet mode
-	r.versionChecker = notification.NewVersionChecker(
-		notification.WithSkipVersionCheck(cliOptions.SkipVersionCheck),
-		notification.WithQuietMode(cliOptions.Quiet),
-		notification.WithTelemetryDisabled(cliOptions.DisableTelemetry),
-	)
+	// get the sub command name with fallback to trivy for version check telemetry
+	commandName := "trivy"
+	if len(os.Args) > 1 {
+		commandName = os.Args[1]
+	}
+	r.versionChecker = notification.NewVersionChecker(commandName, &cliOptions)
```

Comment:
nit
a little simpler:
```suggestion
	r.versionChecker = notification.NewVersionChecker(lo.Ternary(len(os.Args) > 1, os.Args[1], "trivy"), &cliOptions)
```

Reply:
yup using this now to get the `commandName`